include/mrdox/Metadata/Typedef.hpp

```diff
@@ -31,15 +36,29 @@ struct TypedefInfo
     //   typedef std::vector<int> MyVector;
     bool IsUsing = false;
 
+    std::unique_ptr<TemplateInfo> Template;
```

Comment:
is it just me or can almost everything be a template? Should we move this data member to a base?

Reply:
Enums cannot be templates